source/fuzz/fuzzer_pass_add_loads.cpp

```diff
@@ -79,13 +83,46 @@ void FuzzerPassAddLoads::Apply() {
           return;
         }
 
-        // Choose a pointer at random, and create and apply a loading
-        // transformation based on it.
-        ApplyTransformation(TransformationLoad(
-            GetFuzzerContext()->GetFreshId(),
+        auto chosen_instruction =
             relevant_instructions[GetFuzzerContext()->RandomIndex(
-                                      relevant_instructions)]
-                ->result_id(),
+                relevant_instructions)];
+
+        bool is_atomic_load = false;
+        uint32_t memory_scope_id = 0;
+        uint32_t memory_semtantics_id = 0;
+
+        // has work group or storage buffer (storage class)
+        if (chosen_instruction->GetInOperand(0).words[0] ==
```

Comment:
Nit regarding the comment style: `// has work group or storage buffer (storage class)`

Use a capital letter at the start and a full stop (period) at the end. Also, this comment will probably change once you have updated the code because there are about 4 storage classes that need to be considered (but you don't need to list the storage classes in the comment AND the code).

Reply:
Removed.